### DISCUSSION TITLE
panic: runtime error: invalid memory address or nil pointer dereference

### DISCUSSION THREAD

**Discussion by ardzz** [projectdiscovery/nuclei#4763] [General]
### Nuclei version:

[INF] Nuclei Engine Version: v2.9.15
### Current Behavior:

[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x47d1a2]

goroutine 314080893 [running]:
io.(*nopCloserWriterTo).Read(0xc040282a80?, {0xc02269b65c?, 0xc044f3c3f0?, 0x487?})
        <autogenerated>:1 +0x22
net/http.(*transferWriter).probeRequestBody.func1({0x7f97bb677640, 0x0})
        net/http/transfer.go:211 +0x6e
created by net/http.(*transferWriter).probeRequestBody
        net/http/transfer.go:208 +0x12e

### Expected Behavior:

No segmentation violation

### Steps To Reproduce:
1. Run `nuclei -l target.txt -et network,dns,ssl,file -eid tech-detect,http-missing-security-headers,waf-detect,cors-misconfig,options-method,robots-txt-endpoint,robots-txt,http-trace,CVE-2020-35489,wordpress-litespeed-cache,wordpress-user-role-editor,wordpress-wp-optimize,azure-domain-tenant,wordpress-user-enum,wordpress-rdf-user-enum,wordpress-readme-file,wordpress-wordpress-seo,wp-license-file,wordpress-xmlrpc-listmethods,wordpress-login,wordpress-detect,wordpress-xmlrpc-file,metatag-cms -o output.txt`
2. See error

**Comment by olearycrew** [comment]
@ardzz the latest version of nuclei is `v3.1.10` - can you try updating and see if you get the same error?

**Reply by ardzz** [reply]
okay, now i'm using the latest version. i'll give u an update, thanks

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
